packages/core/src/api/index.ts

```diff
@@ -28,6 +30,8 @@ export * from './voice.js';
 export * from './webhook.js';
 
 export class API {
+	public readonly application: ApplicationsAPI;
```

Comment:
This should be probably be pluralized to match with the other fields.
```suggestion
	public readonly applications: ApplicationsAPI;
```

Reply:
I thought about this too, but there are many application commands, channels, guilds, interactions, invites, etc. but only _one_ application to do things against. Should I still change it?

Reply:
When I used the plural version, it was more along the lines of "this is the API for {X}s" rather than it being an indication if multiple versions of X are available. I think this and the change of the name to `getCurrent()` would fit better.